scholia/app/templates/base.html

```diff
@@ -35,9 +35,9 @@
 
       try {
           var orcid = item.claims.P496[0].mainsnak.datavalue.value;
-          $( '#details' ).append( '<div><!-- a href="https://orcid.org/"><!-- img alt="' +
-	      'ORCID logo" src="https://orcid.org/sites/default/files/images/orcid_16x16.png"' +
-	      'width="16" height="16" hspace="4" /></a --> <a href="https://orcid.org/' +
+          $( '#details' ).append( '<div><a href="https://orcid.org/"><img alt="' +
+	      'ORCID logo" src="/static/images/orcid_16x16.gif"' +
```

Comment:
I think there is a problem with "/..." due to tools.wmflabs.org having the /scholia prefix.

Reply:
I copied that from another file, just to prevent issues like that :(

Reply:
I'll push a fix...

Reply:
testing the fix...